docs/deployment/postgres-database-deployment.md

```diff
@@ -352,6 +375,95 @@ The content of each of these secrets are in the form:
 - `username` -- the account username
 - `port` -- The database connection port, should always be 5432
 
+</details>
+
+---
+
+### Connect to PostgreSQL DB via pgAdmin
+
+If you would like to manage your PostgreSQL database in an GUI tool you can via pgAdmin.
+
+#### Requirements
+
+- Download pgAdmin: [A management tool for PostgreSQL](https://www.pgadmin.org/)
+- Installation of AWS CLI ([installation steps](https://docs.aws.amazon.com/cli/latest/userguide/getting-started-install.html))
+- Installation of SSM AWS CLI plugin ([installation steps](https://docs.aws.amazon.com/systems-manager/latest/userguide/session-manager-working-with-install-plugin.html))
```

Comment:
```suggestion
- Install AWS CLI ([installation steps](https://docs.aws.amazon.com/cli/latest/userguide/getting-started-install.html))
- Install SSM AWS CLI plugin ([installation steps](https://docs.aws.amazon.com/systems-manager/latest/userguide/session-manager-working-with-install-plugin.html))
```

Not sure if this is a better improvement but it's consistent with the first bullet point 😅

Reply:
Yeah, I'm okay with the redundancy. That makes sense to have that in the intro as a lead into the steps. Thanks. 😀